src/aind_data_access_api/document_db.py

```diff
@@ -122,15 +122,17 @@ def _count_records(self, filter_query: Optional[dict] = None):
         if filter_query is not None:
             params["filter"] = json.dumps(filter_query)
         response = requests.get(self._base_url, params=params)
-        response_json = response.json()
-        body = response_json.get("body")
-        return json.loads(body)
+        if response.status_code != 200:
+            error_msg = response.text if response.text else "Unknown error"
+            raise ValueError(f"{response.status_code} Error: {error_msg}")
+        response_body = response.json()
+        return response_body
 
     def _get_records(
         self,
         filter_query: Optional[dict] = None,
         projection: Optional[dict] = None,
-        sort: Optional[List[Tuple[str, int]]] = None,
+        sort: Optional[dict] = None,
```

Comment:
This was changed because the nodejs mongo driver requires a dict for sort keys rather than list